packages/config/hackkit.config.ts

```diff
@@ -861,10 +861,10 @@ const c = {
 		guide: "https://go.rowdyhacks.org/discord",
 	},
 	icon: {
-		sm: "/img/logo/hackkit.svg",
-		md: "/img/logo/hackkit-md.png",
-		lg: "/img/logo/hackkit-lg.png",
-		svg: "/img/logo/hackkit.svg",
+		sm: "/img/logo/rhbttf.svg",
```

Comment:
question: did you check to see if all of these files exist where you say they do?

Reply:
yes they do

Reply:
the ones on lines 865 and 866 do not exist

Reply:
should i request the design team to make them?